core/src/banking_stage.rs

```diff
@@ -492,16 +489,12 @@ impl BankingStage {
         packet_receiver: &mut PacketReceiver,
         decision_maker: &DecisionMaker,
         forwarder: &Forwarder,
-        committer: &Committer,
-        poh_recorder: &Arc<RwLock<PohRecorder>>,
+        consumer: &Consumer,
         id: u32,
-        log_messages_bytes_limit: Option<usize>,
         mut unprocessed_transaction_storage: UnprocessedTransactionStorage,
     ) {
-        let recorder = poh_recorder.read().unwrap().recorder();
```

Comment:
:byebye: 

yeah, i really didn't like this waste of read contention source to create same object over and over inside hot loop... were you bothered enough to a git log history revealing of the fact how we got in this form? ;)

Reply:
I didn't look into the history of how we got into the situation. I imagine it's because we used the actual `PohRecorder` elsewhere in BankingStage